roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -188,6 +188,7 @@ ifconfig-pool-persist ipp.txt
 # or bridge the TUN/TAP interface to the internet
 # in order for this to work properly).
 ;push "redirect-gateway def1 bypass-dhcp"
+;push "dhcp-option DNS 8.8.8.8"
```

Comment:
we removed this line earlier, don't want to promote services run by 3rd parties unless necessary.